doc/classes/AudioEffectEQ.xml

```diff
@@ -1,11 +1,12 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="AudioEffectEQ" inherits="AudioEffect" category="Core" version="3.2">
 	<brief_description>
-		Base class for audio equalizers. Gives you control over frequencies.
-		Use it to create a custom equalizer if [AudioEffectEQ6], [AudioEffectEQ10] or [AudioEffectEQ21] don't fit your needs.
+		Base class for audio equalizers. Choose among [AudioEffectEQ6], [AudioEffectEQ10], [AudioEffectEQ21] or [AudioEffectEQCustom] according to specific needs.
 	</brief_description>
 	<description>
-		AudioEffectEQ gives you control over frequencies. Use it to compensate for existing deficiencies in audio. AudioEffectEQs are useful on the Master bus to completely master a mix and give it more character. They are also useful when a game is run on a mobile device, to adjust the mix to that kind of speakers (it can be added but disabled when headphones are plugged).
+		An equalizer gives you control over different parts of the audio according to frequencies. It boosts or suppresses certain frequencies (bands) according to the setup.
```

Comment:
I am absolutely not certain, but I think you should use "Related to frequencies" instead of "according to frequencies". Please correct me if I'm wrong.

Reply:
The idea is that you can roughly think of equalizers as splitting the audio into bands with different frequencies, and controlling the volume of each of them. What can be proper for this?

(Thanks for all the suggestions, I will make changes soon!)